en/02_Developer_Guides/01_Templates/01_Syntax.md

```diff
@@ -69,32 +68,33 @@ This will result in trying to fetch a `Title` property from the data being displ
 Variables can be chained together, and include arguments.
 
 ```ss
-$Foo(param)
+$Foo("param")
 $Foo.Bar
 ```
 
 These variables will call a method / field on the object and insert the returned value as a string into the template.
 
+Arguments don't have to be literals - you can pass template variables as arguments as well.
```

Comment:
This is new and was mostly just a natural consequence of how we're wrapping data now. Very useful though, I'm already gonna use it in the CMSMain refactor.